x/stake/types/validator.go

```diff
@@ -114,20 +114,20 @@ func MustUnmarshalValidator(cdc *wire.Codec, ownerAddr, value []byte) Validator
 }
 
 // unmarshal a redelegation from a store key and value
-func UnmarshalValidator(cdc *wire.Codec, ownerAddr, value []byte) (validator Validator, err error) {
+func UnmarshalValidator(cdc *wire.Codec, operatorAddr, value []byte) (validator Validator, err error) {
 	var storeValue validatorValue
 	err = cdc.UnmarshalBinary(value, &storeValue)
 	if err != nil {
 		return
 	}
 
-	if len(ownerAddr) != 20 {
+	if len(operatorAddr) != 20 {
```

Comment:
magic number

Reply:
I believe @melekes is suggesting this should be a constant -- correct me if I'm wrong ;-)

Reply:
Yes, I think he is. @melekes could you elaborate please?

Reply:
What @alexanderbez said ^^